### DISCUSSION TITLE
Possibility to add a Maximum and/or Minimum Video Quality

### DISCUSSION THREAD

**Discussion by tonogdlp** [f1multiviewer/issue-tracker#11] [Feature requests]
First of all, awesome job mate! This is already a great app!

I have a somewhat limited bandwith, so I would like to have a better way to manage my feeds' video quality by setting a max and/or min bitrate. 

For example, if 2 roomates go to a zoom call at the same time, and my bandwith starts to suffer, I obviously rather have the best possible quality in the race feed, and maybe let some of the On Boards suffer a little bit. So I could set all Onboards to Auto, and my race feed to 'Min: 720p'.

Also, I usually have the Driver Tracker in a tiny window.. While I do want it to look good, it is wasteful to put it on "Auto", if it means that it will gobble up bandwith needlessly.. But, at the same time I don't wan't to set it to a specific bitrate in case my internet speed suffers, and it needs to degrade the quality or else I risk the feed stopping something else.. 

I feel that it would be something of a stretch, but I will also would love if there was a 'Prioritize this feed quality' button or something like that

**Comment by jeppeolesen** [comment]
hey @tonogdlp 

Closing this issue as wont fix, mostly because the "Quality" selector in the player has the auto option which balances the quality based off of the available bandwidth.

Re. the driver tracker, if you want to minimise bandwidth usage, using the MultiViewer Track Map in the live timing is the way to go.
